server-spi-private/src/main/java/org/keycloak/authorization/store/PolicyStore.java

```diff
@@ -92,7 +95,13 @@
      * @param resourceServerId the resource server id
      * @return a list of policies associated with the given resource
      */
-    List<Policy> findByResource(String resourceId, String resourceServerId);
+    default List<Policy> findByResource(String resourceId, String resourceServerId) {
+        List<Policy> result = new LinkedList<>();
+
+        findByResource(resourceId, resourceServerId, result::add);
+
+        return result;
+    }
 
     void findByResource(String resourceId, String resourceServerId, Consumer<Policy> consumer);
```

Comment:
It might be worth to add javadoc for this method as well.

Reply:
It might be worth adding but it wasn't supposed to be part of this Jira. If I remember correctly we decided to just provide map storage and not do cleanup as for the other entities. @hmlnarik Can you confirm? Should I create another task for cleanup of interfaces or should I add Javadoc as part of this one?